apps/cowswap-frontend/src/utils/orderUtils/calculateRateForAmount.ts

```diff
@@ -0,0 +1,24 @@
+import { Currency, CurrencyAmount, Price } from '@uniswap/sdk-core'
+
+export function calculateRateForAmount(
```

Comment:
Could you please add a unit test for this function?
Not merge blocking

Reply:
Will do

Reply:
https://github.com/cowprotocol/cowswap/pull/5339